src/com/google/enterprise/adaptor/documentum/DocumentumAdaptor.java

```diff
@@ -453,6 +458,9 @@ public void init(AdaptorContext context) throws DfException {
     try {
       validateStartPaths(dmSession);
       validateDocumentTypes(dmSession);
+      if (!Strings.isNullOrEmpty(updatedDocumentsQuery)) {
```

Comment:
Use isEmpty. It can't be null. Or, set it to null in addKey, and check for isNullOrEmpty below. Might have the advantage of not needing to set it in the tests unless we're using it.